src/raft.rs

```diff
@@ -1811,14 +1993,63 @@ impl<T: Storage> Raft<T> {
                 };
                 self.read_states.push(rs);
             }
+            MessageType::MsgAppendResponse => {
+                if self.prs().get(m.from).is_none() {
```

Comment:
It's a common case for receiving responses from removed peers. So I think the log is unnecessary. Without it the code could be cleaner.